jsonargparse_tests/test_typehints.py

```diff
@@ -693,6 +696,44 @@ def test_callable_args_return_type_class(parser, subtests):
             assert f"{__name__}.{name}" in help_str
 
 
+def test_callable_multiple_args_return_type_class(parser, subtests):
+    parser.add_argument("--optimizer", type=Callable[[List[float], float], Optimizer], default=SGD)
```

Comment:
This looks good! Since it is just for testing, then there is no real risk that people use this as reference to learn the concept.